__tests__/app.test.js

```diff
@@ -5,9 +5,134 @@ const data = require('../db/data/test-data')
 const db = require('../db/connection')
 const endpoints = require('../endpoints.json')
 
-beforeEach(() => seed(data));
+beforeEach(() => seed(data))
 afterAll(() => db.end());
 
+describe('PATCH', () => {
+    describe('/api/articles/:article_id', () => {
+        test('PATCH: 200 - update the votes and responds with updated article', () => {
+            return request(app)
+                .patch('/api/articles/1')
+                .send({ inc_votes: 10 })
+                .expect(200)
+                .then((response) => {
+                    const { article } = response.body  
+                    expect(article).toHaveProperty('votes', expect.any(Number))
+                    expect(article.votes).toBeGreaterThan(0)
```

Comment:
why not test for the exact number of votes the article would have?
If the article already some votes but your patch wasn't working you would still have it pass